build.sbt

```diff
@@ -464,11 +464,14 @@ lazy val reporters = (project in file("reporters"))
 
 
 lazy val `kamon-datadog` = (project in file("reporters/kamon-datadog"))
-  .disablePlugins(AssemblyPlugin)
+  .enablePlugins(AssemblyPlugin)
```

Comment:
Hmm, I'm not sure if this is something we can change, we'll wait for input from @ivantopo

Reply:
Yeah I wasn't sure either. But, it won't publish without it (I tried locally). The shading stuff relies on that plugin, looks like, so I follwed `kamon-status-page`'s lead.